wordpress-mobile/WordPress-iOS
# UL&S: Navigate to blank site address view controller

Ref. #14009 
Ref. https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/293

In this PR we're navigating to the new Site Address flow. The only thing marking the new SiteAddressViewController is a temporary label displaying the class name.

### To test:
**Test Case 1: unified site address enabled**
1. Check out this branch
2. `rake dependencies`
3. Build and run
4. Log In > "Or log in by entering your site address."

**Expected:** you are navigated to the new site address controller.
<kbd><a href="https://user-images.githubusercontent.com/1062444/83439611-20ca2100-a409-11ea-8d9d-5e04d615747c.png"><img src="https://user-images.githubusercontent.com/1062444/83439611-20ca2100-a409-11ea-8d9d-5e04d615747c.png" width="300" /></a></kbd>

**Test Case 2: old self-hosted flow on email login displays**
1. Navigate back if you're still in the new site address view controller.
2. Log In > "Continue with WordPress.com" > "Or log in by entering your site address."
**Expected:** you are navigated to the old site address controller.
**Note:** this navigates you to the old flow because logging in after entering the email login flow won't exist in the new site address flow. So there is no reason to navigate to the new Site Address view controller from here.

**Test Case 3: unified auth disabled**
1. Disable the [unified auth feature flag](https://github.com/wordpress-mobile/WordPress-iOS/blob/feature/14009-unify-site-address-i/WordPress/Classes/Utility/BuildInformation/FeatureFlag.swift#L30)
2. Build and run
3. Log In > "Or log in by entering your site address."
**Expected:** you are navigated to the old self-hosted controller.

PR submission checklist:
